uber/site_sections/registration.py

```diff
@@ -153,6 +153,7 @@ def change_badge(self, session, id, message='', **params):
             'attendee': attendee
         }
 
+    @unrestricted
```

Comment:
actually /uber/registration/register uses ```@unrestricted``` as well.

but yea might be a decent idea to consolidate this stuff in the future